README.md

```diff
@@ -241,4 +241,35 @@ The command will start the container but will not run Flask. To run the app in d
 7. Select Flask from dropdown menu that appears
 8. Type `todo_app/app.py` into the input box that appears
 
- 
\ No newline at end of file
+## Deployment to Azure 
+
+The application is deployed through Azure App Service using Docker. New changes will need to be pushed to Docker Hub.
```

Comment:
Minor, but it would be nice to mention the address at which this is hosted!
<https://cb-todo-app.azurewebsites.net/>